app/components/attachment_button.js

```diff
@@ -65,12 +65,64 @@ export default class AttachmentButton extends PureComponent {
         intl: intlShape.isRequired,
     };
 
+    getPermissionDeniedMessage = (source) => {
+        const {formatMessage} = this.context.intl;
+        switch (source) {
+        case 'camera':
+            return {
+                title: formatMessage({
+                    id: 'mobile.android.camera_permission_denied_title',
+                    defaultMessage: 'Camera access is required',
+                }),
+                text: formatMessage({
+                    id: 'mobile.android.camera_permission_denied_description',
+                    defaultMessage: 'To take photos and videos with your camera, please change your permission settings.',
```

Comment:
Please update your permission settings to take photos and videos with your camera.